Tests/MessagingInApp/EngineWeb/EngineWebProviderStub.swift

```diff
@@ -12,17 +12,3 @@ class EngineWebProviderStub: EngineWebProvider {
         return newMock
     }
 }
-
-// In order to test multiple instances of inline Views, you need to create multiple separate EngineWebInstance instances for each View instance.
-// This is a new stub that allows this.
-// A follow-up refactor that makes this stub the default would be a good chance to the test suite.
-class EngineWebProviderStub2: EngineWebProvider {
```

Comment:
This class was removed in feature branch. Not needed, so I am removing it again.